projects/plugins/jetpack/_inc/client/at-a-glance/backup-upgrade/types.ts

```diff
@@ -0,0 +1,17 @@
+export interface BackupUpgradeBaseProps {
```

Comment:
This should be named `BarChartBaseProps` since it's the current one that uses these values. `BackupUpgrade` uses it only for analytics and passes it down.

Reply:
Good one. Thanks